docs/source/noise/index.rst

```diff
@@ -110,60 +110,60 @@ Noise types for each column
   :header-rows: 1
 
   * - Column name
-    - Datasets present
+    - Applicable datasets
     - Types of noise
     - Notes
   * - First name
     - Decennial Census, ACS, CPS, WIC, W-2 and 1099, 1040, SSA
-    - Leave a field blank, use a fake name, use a nickname, make typos, make OCR errors, make phonetic errors
+    - Leave a field blank, use a nickname, use a fake name, make phonetic errors, make OCR errors, make typos
     - In the 1040 form, the same noise types apply to the first name columns for the joint filer and dependents
   * - Middle name
     - SSA
-    - Leave a field blank, use a fake name, use a nickname, make typos, make OCR errors, make phonetic errors
-    - 
+    - Leave a field blank, use a nickname, use a fake name, make phonetic errors, make OCR errors, make typos
+    - Middle names use the same lists of nicknames and fake names used for first names
```

Comment:
I thought it would be good to specify that middle names use the same data as first names for these two noise functions. This is unrelated to the noise function order.